posts/2021-09-17-binding-app-to-postgresql-db-with-odo-and-ol-stack.adoc

```diff
@@ -151,7 +116,7 @@ metadata:
     service.binding/db_user: 'path={.spec.databaseUser}'
 ----
 
-Adding the annotations ensures that the Service Binding Operator will inject the `databaseName`, `databasePassword` and `databaseUser` spec values into the application. Note that the instance name you configure will be used as part of the name of various artifacts and resource references. Be sure to change it.
+Adding the annotations ensures that odo will inject the `databaseName`, `databasePassword` and `databaseUser` spec values into the application. Note that the instance name you configure will be used as part of the name of various artifacts and resource references. Be sure to change it.
```

Comment:
I think we need to at least make a nod to the fact that there's a mapping happening under the covers...before by SBO, now by odo itself, to get to  server.xml vars like:
`${DATABASE_DB_USER}`